pydantic_core/_pydantic_core.pyi

```diff
@@ -178,15 +180,22 @@ class SchemaError(Exception):
     def error_count(self) -> int: ...
     def errors(self) -> 'list[ErrorDetails]': ...
 
+Location = tuple[str | int, ...]
+
 class ValidationError(ValueError):
     def __init__(
-        self, title: str, errors: 'list[InitErrorDetails]', error_mode: Literal['python', 'json'] = 'python'
+        self,
+        title: str,
+        errors: 'list[InitErrorDetails | ValidationError]',
+        error_mode: Literal['python', 'json'] = 'python',
+        loc_prefix: Location | None = None,
     ) -> None: ...
     @property
     def title(self) -> str: ...
     def error_count(self) -> int: ...
     def errors(self, *, include_url: bool = True, include_context: bool = True) -> 'list[ErrorDetails]': ...
     def json(self, *, indent: 'int | None' = None, include_url: bool = True, include_context: bool = False) -> str: ...
+    def derive(self: Self, __exceptions: 'list[InitErrorDetails | ValidationError]') -> Self: ...
```

Comment:
This feels to me like it wants to be a classmethod, I'm guessing that's hard to do from rust?

Reply:
No it’s specified like this in the PEP

Reply:
lol okay, goes to show me.

Reply:
https://peps.python.org/pep-0654/ in case anyone else sees this